packages/lesswrong/components/form-components/FormComponentPostEditorTagging.tsx

```diff
@@ -102,23 +144,31 @@ const FormComponentPostEditorTagging = ({value, path, document, formType, update
   } else {
     return (
       <div className={classes.root}>
-        {showCoreTopicSection && (
+        {showCoreAndTypesTopicSections && (
           <>
             <h3 className={classNames(classes.coreTagHeader, classes.header)}>Core topics</h3>
             <TagsChecklist
-              tags={coreTagsToDisplay}
+              tags={coreTags}
+              selectedTagIds={selectedTagIds}
+              onTagSelected={onTagSelected}
+              onTagRemoved={onTagRemoved}
+              displaySelected="highlight"
+            />
+            <h3 className={classNames(classes.coreTagHeader, classes.header)}>Common post types</h3>
```

Comment:
Maybe this should not show when there are no `postTypeTags`, which will be the case for a few minutes after we deploy